xml/admin_rbd.xml

```diff
@@ -1030,17 +1027,49 @@ EOF</screen>
 <screen>&prompt.cephuser;rbd --cluster remote mirror pool peer bootstrap import \
  --site-name remote image-pool token</screen>
    </sect3>
-   <sect3>
-    <title>Add Cluster Peer</title>
+   <sect3 xml:id="ceph-rbd-mirror-add-peer">
+    <title>Add Cluster Peer Manually</title>
     <para>
-     In order for the <systemitem>rbd-mirror</systemitem> daemon to discover
-     its peer cluster, the peer needs to be registered to the pool. To add a
-     mirroring peer cluster, specify the <command>mirror pool peer
-     add</command> subcommand, the pool name, and a cluster specification:
+     Alternatively to bootstrapping peers as described in <xref
+     linkend="ceph-rbd-mirror-bootstrap-peer"/>, you can specify peers
+     manually. The remote &rbdmirror; daemon will need access to the local
+     cluster to perform mirroring. Create a new local &ceph; user that the
+     remote &rbdmirror; daemon will use, for example
+     <literal>rbd-mirror-peer</literal>:
     </para>
 <screen>
-&prompt.cephuser;rbd --cluster local mirror pool peer add <replaceable>POOL_NAME</replaceable> client.rbd-mirror-peer@remote
-&prompt.cephuser;rbd --cluster remote mirror pool peer add <replaceable>POOL_NAME</replaceable> client.rbd-mirror-peer@local
+&prompt.cephuser;ceph auth get-or-create client.rbd-mirror-peer \
+ mon 'profile rbd' osd 'profile rbd'
+</screen>
+    <para>
+     Use the following syntax to add a mirroring peer &ceph; cluster with the
+     <command>rbd</command> command:
+    </para>
+<screen>rbd mirror pool peer add <replaceable>POOL_NAME</replaceable> <replaceable>CLIENT_NAME</replaceable>@<replaceable>CLUSTER_NAME</replaceable></screen>
+    <para>
+     For example:
+    </para>
+<screen>
+&prompt.cephuser;rbd --cluster site-a mirror pool peer add image-pool client.rbd-mirror-peer@site-b
+&prompt.cephuser;rbd --cluster site-b mirror pool peer add image-pool client.rbd-mirror-peer@site-a
+</screen>
+   <para>
+    The &rbdmirror; daemon needs access to the peer cluster’s MON IP addresses
+    and client key. <command>rbd</command> uses the
+    <literal>config-key</literal> store to save this information. To specify
```

Comment:
This paragraph is not very clear to me. "needs access to IP addresses" sounds wrong. It needs access to the cluster. Also it does not specify that the config-key store is an alternative to providing a config file where the rbd-mirror would try to find the information first.

Why not just copy&past what we have upstream [1]:
```
By default, the rbd-mirror daemon needs to have access to a Ceph configuration file located at /etc/ceph/{cluster-name}.conf that provides the addresses of the peer cluster’s monitors, in addition to a keyring for {client-name} located in the default or configured keyring search paths (e.g. /etc/ceph/{cluster-name}.{client-name}.keyring).

Alternatively, the peer cluster’s monitor and/or client key can be securely stored within the local Ceph monitor config-key store. To specify the peer cluster connection attributes when adding a mirroring peer, use the --remote-mon-host and --remote-key-file optionals. 
```
It looks much clearer to me.

[1] https://docs.ceph.com/en/octopus/rbd/rbd-mirroring/#add-cluster-peer-manually